docs/dev/table/sqlClient.md

```diff
@@ -0,0 +1,538 @@
+
+Properties that have been set within a CLI session (e.g. using the `SET` command) have highest precedence:
+
+{% highlight text %}
+CLI commands > session environment file > defaults environment file
+{% endhighlight %}
+
+{% top %}
+
+### Dependencies
+
+The SQL Client does not require to setup a Java project using Maven or SBT. Instead, you can pass the dependencies as regular JAR files that get submitted to the cluster. You can either specify each JAR file separately (using `--jar`) or define entire library directories (using `--library`). For connectors to external systems (such as Apache Kafka) and corresponding data formats (such as JSON), Flink provides **ready-to-use JAR bundles**. These JAR files are suffixed with `sql-jar` and can be downloaded for each release from the Maven central repository.
+
+
+
+#### Connectors
+
+| Name              | Version       | Download               |
+| :---------------- | :------------ | :--------------------- |
+| Filesystem        |               | Built-in               |
+| Apache Kafka      | 0.8           | [Download](http://central.maven.org/maven2/org/apache/flink/flink-connector-kafka-0.8{{site.scala_version_suffix}}/{{site.version}}/flink-connector-kafka-0.8{{site.scala_version_suffix}}-{{site.version}}-sql-jar.jar) |
+| Apache Kafka      | 0.9           | [Download](http://central.maven.org/maven2/org/apache/flink/flink-connector-kafka-0.9{{site.scala_version_suffix}}/{{site.version}}/flink-connector-kafka-0.9{{site.scala_version_suffix}}-{{site.version}}-sql-jar.jar) |
+| Apache Kafka      | 0.10          | [Download](http://central.maven.org/maven2/org/apache/flink/flink-connector-kafka-0.10{{site.scala_version_suffix}}/{{site.version}}/flink-connector-kafka-0.10{{site.scala_version_suffix}}-{{site.version}}-sql-jar.jar) |
+| Apache Kafka      | 0.11          | [Download](http://central.maven.org/maven2/org/apache/flink/flink-connector-kafka-0.11{{site.scala_version_suffix}}/{{site.version}}/flink-connector-kafka-0.11{{site.scala_version_suffix}}-{{site.version}}-sql-jar.jar) |
+
+#### Formats
+
+| Name              | Download               |
+| :---------------- | :--------------------- |
+| CSV               | Built-in               |
+| JSON              | [Download](http://central.maven.org/maven2/org/apache/flink/flink-json/{{site.version}}/flink-json-{{site.version}}-sql-jar.jar) |
+
+
+{% top %}
+
+Table Sources
+-------------
+
+Sources are defined using a set of [YAML properties](http://http://yaml.org/). Similar to a SQL `CREATE TABLE` statement you define the name of the table, the final schema of the table, connector, and a data format if necessary. Additionally, you have to specify it's type (source, sink, or both).
+
```

Comment:
For required/optional presentation conventions we also have sth here https://ci.apache.org/projects/flink/flink-docs-release-1.4/dev/stream/operators/windows.html . It may make sense to try to uniformize. This is also for below.